src/sentry/demo_mode/tasks.py

```diff
@@ -0,0 +1,127 @@
+from datetime import timedelta
+
+from django.db.models import Q
+from django.utils import timezone
+
+from sentry import options
+from sentry.models.artifactbundle import (
+    ArtifactBundle,
+    ProjectArtifactBundle,
+    ReleaseArtifactBundle,
+)
+from sentry.models.organization import Organization
+from sentry.models.project import Project
+from sentry.tasks.base import instrumented_task
+from sentry.utils.demo_mode import get_demo_org, is_demo_mode_enabled
+
+
+@instrumented_task(
+    name="sentry.demo_mode.tasks.sync_artifact_bundles",
+    queue="demo_mode",
+)
+def sync_artifact_bundles():
+
+    if (
+        not options.get("sentry.demo_mode.sync_artifact_bundles.enable")
+        or not is_demo_mode_enabled()
+    ):
+        return
+
+    source_org_id = options.get("sentry.demo_mode.sync_artifact_bundles.source_org_id")
+    source_org = Organization.objects.get(id=source_org_id)
+
+    target_org = get_demo_org()
+
+    lookback_days = options.get("sentry.demo_mode.sync_artifact_bundles.lookback_days")
+
+    _sync_artifact_bundles(source_org, target_org, lookback_days)
+
+
+def _sync_artifact_bundles(source_org: Organization, target_org: Organization, lookback_days=1):
```

Comment:
I would recommend wrapping all of the code in a transaction that spans all the tables, since we want to avoid partial state in case of task failure.

```python
with atomic_transaction(
                        using=(
                            router.db_for_write(ArtifactBundle),
                            router.db_for_write(ProjectArtifactBundle),
			   router.db_for_write(ReleaseArtifactBundle),
                        )
                    ):
	# all the code
```